README.rst

```diff
@@ -340,9 +336,10 @@ like 'radosgw-agent'. This inclusion would be defined at the project level, like
     repos = {
         'ceph': {
             'extras': ['ceph-deploy', 'radosgw-agent'],
-            'deb': {
```

Comment:
*however*, we could also keep `'combined': True` and make them *all* the ones available for the DEBs be combined.

When/why/where would it make sense to have certain distro versions combined and not the rest of them? With this key being a boolean is either you get them all combined or not.